### PR TITLE
2021-06-29 Release

### PR DESCRIPTION
**Pull request by shawntanzk**
Quick release to update CL for a BDS workshop

Edits:
- CL:000411 - changed from flatworm hypodermal cell -> Caenorhabditis hypodermal cell
- tears secreting cell no subset of CL:0000152 ! exocrine cell
- shorten names of various neuronal types
- Added BDS subset tags to additional cell types

### CODE REVIEW

File: cl-base.obo [hunk 1]
```diff
@@ -523,18 +523,6 @@ synonym: "mature eosinophil leukocyte" EXACT []
...
-intersection_of: BFO:0000051 PR:000007152
-intersection_of: BFO:0000051 PR:000007857
-intersection_of: BFO:0000051 PR:000007858
-intersection_of: BFO:0000051 PR:000014046
-intersection_of: BFO:0000051 PR:000014047
-intersection_of: BFO:0000051 PR:000022006
-intersection_of: RO:0002104 PR:000009129
-intersection_of: RO:0002215 GO:0002495
-intersection_of: RO:0002215 GO:0002540
-intersection_of: RO:0002215 GO:0045730
```
Comment (dosumis) [inline cl-base.obo:537]: Any idea where this was deleted? Was it intentional?

**Review by dosumis — COMMENTED** [review]
Investigate origin deletion EC axioms on mature eosinophil and mature basophil

File: cl-base.obo (hunk 1 above)
Comment (shawntanzk) [reply cl-base.obo:537]: not quite sure - definitely has something to do with [obsoleted lobed nucleus term](https://github.com/obophenotype/cell-ontology/issues/1130), but I would have thought that would change in [last release](https://github.com/obophenotype/cell-ontology/pull/1152) that @cmungall did ytd?

Comment (shawntanzk) [reply cl-base.obo:537]: Screenshot 2021-06-29 at 11 43 28
Think this is it - was in cl.obo in the prerelease ytd

**Review by dosumis — APPROVED** [review]

File: cl-base.obo (hunk 1 above)
Comment (shawntanzk) [reply cl-base.obo:537]: I've also checked in protege and everything looks fine

**Pull request merged**